src/core/sys/posix/setjmp.d

```diff
@@ -125,6 +128,18 @@ version (CRuntime_Glibc)
                 double[6] __fpregs;
         }
     }
+    else version (RISCV_Any)
+    {
+        struct __riscv_jmp_buf
+        {
+            c_long __pc;
+            c_long[12] __regs;
+            c_long __sp;
+            static if (__traits(getTargetInfo, "floatAbi") == "double")
+                double[12] __fpregs;
```

Comment:
@TurkeyMan - First user of `__traits(getTargetInfo)` ! (ABI models of RISC-V are "soft", "single", "double", and "quad" https://github.com/riscv/riscv-c-api-doc/blob/master/riscv-c-api.md#preprocessor-definitions :-)

Reply:
Yay! We should support basically everything we can behind `getTargetInfo`! It's a nice clean tidy and centralised way to get at target info!